content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -17,9 +17,6 @@ preferred package manager client.
 
 For example, using the .NET CLI:
 
-```shell
-dotnet add package OpenTelemetry
-```
```

Comment:
Removing this makes the previous line confusing.

To make this kind of change, you should also remove that line, and then below where `OpenTelemetry.Exporter.Console` is installed, mention that it will transitively pull in `OpenTelemetry`.

Reply:
@MujanMujan please give this a look and update your code, thanks!